.github/workflows/lint-test.yaml

```diff
@@ -82,6 +82,12 @@ jobs:
           - name: Horizontal Pod Autoscaling Enabled
             helm_args: '--helm-extra-set-args "--set=hpa.enabled=true --set=hpa.minPods=2 --set=hpa.maxPods=3 --set=hpa.targetCPUUtilizationPercentage=75"'
 
+          # test the helm chart with s3 as the primary storage
+          - name: S3 Enabled as Primary Storage
+            # we need to skip the clean up so we can test adding a file
+            helm_args: |
+                --namespace nextcloud --skip-clean-up --helm-extra-set-args "--set=fullnameOverride=nextcloud --set=nextcloud.objectStore.s3.enabled=true --set=nextcloud.objectStore.s3.accessKey=nextcloud --set=nextcloud.objectStore.s3.secretKey=rootpass123 --set=nextcloud.objectStore.s3.host=minio.nextcloud.svc.cluster.local --set=nextcloud.objectStore.s3.port=9000 --set=nextcloud.objectStore.s3.ssl=false --set=nextcloud.objectStore.s3.bucket=nextcloud --set=nextcloud.objectStore.s3.usePathStyle=true --set=image.flavor=fpm --set=nginx.enabled=true --set=nextcloud.host=nextcloud --set=nextcloud.trustedDomains[0]='*'"
```

Comment:
Any reason this uses nginx? I'd prefer sticking to the default values if it is not necessary to change them for the test (just to be able to isolate problems better in case there are any)

Reply:
Hm I see